arrays/array_practice.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+#Create an array variable with the the numbers 1 through 10
+array = (1..10).to_a
+# 1...2...3...4...5...6...7...8...9...10...
+puts "#{array.join('...')}" + "..."
+# T-10, 9, 8, 7, 6, 5, 4, 3, 2, 1...  BLASTOFF!
+puts 'T-' + array.reverse.to_s.tr('[]',"") + '... BLASTOFF!'
+# The last element is 10
+puts "The last element is #{array.last}"
+# The first element is 1
+puts "The first element is #{array.first}"
+# The third element is 3
+puts "The third element is #{array[2]}"
+# The element with an index of 3 is 4
+puts "The element with an index of #{array[2]} is #{array[3]}"
+# The second from last element is 9
+puts "The second last element is #{array[8]}"
+# The first four elements are '1, 2, 3, 4'
+puts "The first four elements are '#{array.take(4).to_s.tr('[]',"")}'"
+# If we delete 5, 6 and 7 from the array, we're left with [1,2,3,4,8,9,10]
+puts "If we delete #{array.delete_at(4)}, #{array.delete_at(4)} and #{array.delete_at(4)} from the array,  we're left with #{array} "
```

Comment:
This is a valid method, but could the code could be a little cleaner if you use `delete_if`, like so:

```
"...delete 5, 6, and 7 from the array, we're left with #{array.delete_if { # some code }"